src/CFamily.UnitTests/Rules/RulesConfigFixupTests.cs

```diff
@@ -30,6 +33,19 @@ namespace SonarLint.VisualStudio.CFamily.Rules.UnitTests
     [TestClass]
     public class RulesConfigFixupTests
     {
+        private static readonly IHotspotAnalysisConfiguration HotspotsEnabled;
+        private static readonly IHotspotAnalysisConfiguration HotspotsDisabled;
+        
+        static RulesConfigFixupTests()
+        {
+            var hotspotsEnabledMock = new Mock<IHotspotAnalysisConfiguration>();
+            hotspotsEnabledMock.Setup(x => x.IsEnabled()).Returns(true);
+            HotspotsEnabled = hotspotsEnabledMock.Object;
+            var hotspotsDisabledMock = new Mock<IHotspotAnalysisConfiguration>();
+            hotspotsDisabledMock.Setup(x => x.IsEnabled()).Returns(false);
+            HotspotsDisabled = hotspotsDisabledMock.Object;
```

Comment:
This would be easier to read and more consistent with other tests if it had a simple helper method rather than a `static` initializer i.e.

```
private static IHotspotAnalysisConfiguration CreateHotspotAnalysisConfig(bool isEnabled) 
{
  ...
}

```